app/views/users/_organizations_area.html.erb

```diff
@@ -1,18 +0,0 @@
-<%# it's highly unlikely a single use will belong to so many organizations to slow down
-    this fetch, thus we can safely use the .present?/.each pattern to load them %>
```

Comment:
can we keep this comment? I saw it has been removed when you moved over the code